Sources/Orbit/Foundation/Icons/Icons.swift

```diff
@@ -177,243 +140,380 @@ public extension Icon {
+        case messagesOutline
+        case minus
+        case minusCircle
+        case money
+        case moneyTransferIn
+        case moneyTransferOut
+        case moon
+        case musicalInstruments
+        case newWindow
+        case nonstop
+        case notification
+        case notificationAdd
+        case notificationOff
+        case notificationOn
         case onlineCheckin
         case onlineCheckinOff
+        case outlook
+        case paid
+        case parking
+        case partners
         case passenger
-        case passengerOutline
         case passengerAdd
+        case passengerOutline
+        case passengerRemove
         case passengers
         case passport
-        case priorityBoard
+        case pet
+        case pharmacy
+        case phone
+        case playground
+        case plus
+        case plusCircle
+        case pool
+        case powerPlug
+        case powerPlugOff
+        case priorityBoarding
+        case profit
+        case promoCode
+        case qrCode
+        case questionCircle
+        case radar
         case radiusSearch
+        case refund
+        case relax
+        case reload
+        case remove
+        case replace
+        case restaurant
         case routeNoStops
         case routeOneStop
         case routeTwoStops
+        case search
+        case seat
+        case seatAisle
+        case seatExtraLegroom
+        case seatWindow
+        case security
+        case selfTransfer
+        case send
+        case settings
+        case share
+        case shareAndroid
+        case shareIos
+        case shopping
+        case showLess
+        case showMore
         case sightseeing
-        case suitcase
+        case smoking
+        case smokingOff
         case sort
+        case spa
+        case sportEquipment
+        case sports
+        case starEmpty
+        case starFull
+        case subway
+        case suitcase
+        case sun
+        case sunrise
+        case taxi
         case terminal
+        case termsAndConditions
+        case thumbDown
+        case thumbUp
         case ticket
         case ticketOutline
+        case timelapse
+        case timer
         case tips
+        case toilets
+        case train
+        case transmission
         case trip
-        case seatWindow
-        case seatExtraLegroom
-        case seatAisle
-        case seatSitTogether
-
+        case twitter
+        case uber
+        case upload
+        case userGroup
+        case visa
+        case visibility
+        case visibilityOff
+        case walk
+        case wallet
+        case wheelchair
+        case wifi
+        case wifiOff
         case none
 
-        case lock
-        case lockOpen
-
-        case arrowUp
-        case arrowRight
-
-        case billingDetails
-        case customerSupport
-        case security
-
-        case attachment
-        case send
-
-        public var value: String {
+        var value: String {
```

Comment:
@PavelHolec  I made this `internal` - are we depending on this somewhere in the app?